fullnode/src/main.rs

```diff
@@ -32,11 +32,18 @@ fn main() {
                 .help("File containing an identity (keypair)"),
         )
         .arg(
-            Arg::with_name("staker_keypair")
-                .long("staker-keypair")
+            Arg::with_name("staking_account")
+                .long("staking-account")
+                .value_name("PUBKEY_B58_STR")
```

Comment:
nit: `PUBKEY_BASE58_STRING` -- don't make me guess!

Reply:
We just call this plain old "PUBKEY" everywhere else